data/human/sales.txt

```diff
@@ -140,6 +140,56 @@ shipyard "Luxury Ships"
 	"Star Queen"
 	"Flivver"
 
+# Note on Humanika Rafeki Mimicry.
+# For a few decades, New Tortuga has had advanced technology, Syndicate defectors, and a steady supply of poached employees. One of their top priorities has been technological advancement. They are only one planet though, so even with help from an advanced species, their success has been limited. Thus, this list is restricted low-to-mid-range tech from older companies. This is with the exception of the Syndicate, for which they already had access to top technology.
+shipyard "Humanika Rafeki Mimicry"
+	# Advanced Southern Pirates without the Osprey:
+	"Falcon"
```

Comment:
I don't think the Falcon counts as "low-to-mid-range tech from older companies".

Reply:
It is commonly available on pirate planets.

However, it is also redundant. The Leviathan is better anyway.

Falcon is gone, and I've updated the comment.